textattack/transformations/word_swaps/word_swap_masked_lm.py

```diff
@@ -296,6 +383,50 @@ def _get_transformations(self, current_text, indices_to_modify):
         else:
             raise ValueError(f"Unrecognized value {self.method} for `self.method`.")
 
+    def _get_transformations_many(
+        self, list_of_texts: List[str], list_of_indices_to_modify: List
+    ):
+        list_of_indices_to_modify = [
+            list(indices_to_modify) for indices_to_modify in list_of_indices_to_modify
+        ]
+        if self.method == "bae":
+            list_of_replacement_words = self._bae_replacement_words_many(
+                list_of_texts, list_of_indices_to_modify
+            )
+            list_of_transformed_texts = list()
+            for current_text, replacement_words, indices_to_modify in zip(
+                list_of_texts, list_of_replacement_words, list_of_indices_to_modify
+            ):
+                transformed_texts = []
+                for i in range(len(replacement_words)):
+                    index_to_modify = indices_to_modify[i]
+                    word_at_index = current_text.words[index_to_modify]
+                    for word in replacement_words[i]:
+                        if (
+                            word != word_at_index
+                            and len(utils.words_from_text(word)) == 1
+                        ):
+                            transformed_texts.append(
+                                current_text.replace_word_at_index(
+                                    index_to_modify, word
+                                )
+                            )
+                list_of_transformed_texts.append(transformed_texts)
+            return list_of_transformed_texts
+        else:
+            raise ValueError(
+                f"Unrecognized value {self.method} for `self.method` for get_transformations_many."
+            )
```

Comment:
This function need to be moved to the general transformation.py in order to be called in the augmenter